docs/source/create_channel/create_channel_genesis.md

```diff
@@ -0,0 +1,281 @@
+# Using configtx.yaml to create a channel genesis block
+
+A channel is created by building a channel genesis block that contains the initial configuration of the channel. The **channel configuration** is stored on the ledger, and governs all the subsequent blocks that are added to the channel. The channel configuration specifies which organizations are channel members, the ordering nodes that can create new blocks, and the set of organizations that need to agree to channel updates. A Fabric network is formed when a group of organizations create the genesis block of the system channel that defines the ordering service of the network and the blockchain [consortium](../glossary.html#consortium). Peer organizations then can then use the system channel to create the genesis blocks of application channels. The permission to access the network, or update a channel to change the membership or governance, is encoded in the genesis blocks of the system channel and the application channels of the network.
```

Comment:
It's not just in the genesis block, it's in every configuration block.

Reply:
What I am trying to say is that the initial governance, including the ability to create future channel configurations, is encoded in the genesis block. But your right, that this could be clearer.